shopinvader_customer_special_product/models/product.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    manufactured_for_partner_ids = fields.Many2many(
```

Comment:
s/manufactured_for_partner_ids/saleable_to_partner_ids
Whatever the origin of the product, what is important to us is that it can only be sold to specific customers.